src/components/Api.js

```diff
@@ -0,0 +1,146 @@
+export default class Api {
+  constructor(options) {
+    this.baseUrl = options.baseUrl;
+    this.header = options.header;
+  }
+
+  getInitialCards() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+      headers: {
+        authorization: "6bc22e8b-f205-4c04-b4ee-f39ef81e1654",
+        method: "GET",
+      },
+    })
+      .then((res) => res.json())
+      .then((result) => {
+        return result;
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
It’s better not to use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request and this end is in `index.js` after all blocks `then`. And in real projects blocks `catch` usually open error popups and so on.